pkg/logbridge/logbridge.go

```diff
@@ -0,0 +1,88 @@
+package logbridge
+
+import (
+	"bufio"
+	"io"
+
+	"github.com/square/p2/pkg/logging"
+)
+
+// This type is agnostic of pipes. os.File conforms to io.Reader and io.Writer
```

Comment:
After a quick reading through the types and methods to get an overview of what's happening here, I'm still not quite sure what this comment is referring to.  Could you use this comment section to describe what a `Bridge` is or what it does?  A discussion of `os.File` might be relevant, but not without some context.